Cargo.toml

```diff
@@ -76,8 +76,8 @@ fuel-core-types = { version = "0.23.0", path = "./crates/types", default-feature
 fuel-core-tests = { version = "0.0.0", path = "./tests" }
 fuel-core-xtask = { version = "0.0.0", path = "./xtask" }
 
-# Fuel dependencies
-fuel-vm-private = { version = "0.47.1", package = "fuel-vm", default-features = false }
+# Fuel dependencies 
+fuel-vm-private = { package = "fuel-vm", git = "https://github.com/FuelLabs/fuel-vm", branch = "feature/regenesis_off_chain", default-features = false }
```

Comment:
Not sure what to do about this one. Branched off of the latest release. It seems there are some commits after the release that aren't compatible with `fuel-core` `master`.